docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -661,6 +701,31 @@ Dapps can then leverage the relay execution features to create their own busines
 
 ![LSP6 Key Manager Relay Service](/img/standards/lsp6/lsp6-relay-execution.jpeg)
 
+#### What are you signing?
```

Comment:
I suggest we change this title and we make it as subheading 3

```suggestion
### How to sign relay transactions?
```

So that people can find it more easily via the sidebar ⬇️ 
![image](https://user-images.githubusercontent.com/31145285/217234541-c1cae50f-12b3-435f-8390-25d79d9dd4bf.png)